### PR TITLE
ADD product epics to handbook

### PR DESCRIPTION
**Pull request by j-6**
# Description
Adding the documentation epic to the handbook.

## Type of change
- [x] Documentation (update or new)

# How Has This Been Tested?
Has been proofread and edited.

## Testing Checklist:
# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have requested a review on the Pull Request

### CODE REVIEW

**Comment by j-6** [comment]
@ginge270 can you pelase provide some insight into these failed checks?
Figured it out :) had to apply prettier

**Review by ginge270 — APPROVED** [review]
Awesome job 🥇

File: docs/products/ontrack/project-epics/documentation-epic.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+## Business Value
+
+To ensure that all work is usable,and the decision-making process is visible for future stakeholders, the business would like to retrospectively analyse any gaps in the documentation of our products and produce documentation. This will ensure previous decisions, design, and requirements are easily accessible to all future users, developers, and other stakeholders so relevant context is provided for all work in the future.
+
+## In Scope
+
+Design Documentation
+User documentation (high level)
+User guides (step by step how to)
+Requirement documentation
```
Comment (tancnle) [inline docs/products/ontrack/project-epics/documentation-epic.md]: **issue:** Could we convert this to an unordered list? This is [rendered](https://github.com/thoth-tech/handbook/blob/a077af1c31555ad89fd68f1a586dab9543a50784/docs/products/ontrack/project-epics/documentation-epic.md) as one line at the moment.

File: docs/products/ontrack/project-epics/documentation-epic.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## Out of Scope
+
+New feature documentation
+User research documentation
```
Comment (tancnle) [inline docs/products/ontrack/project-epics/documentation-epic.md]: **issue:** Similar suggestion as above. Could we convert this to an unordered list?

File: docs/products/ontrack/project-epics/documentation-epic.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+## Operations / Support / Training Considerations
+
+Advise project teams of relevant documentation releases
+
+## What are the challenges?
+
+Aligning the existing documentation with current version of product
+
+## Acceptance criteria(Informal)
```
Comment (tancnle) [inline docs/products/ontrack/project-epics/documentation-epic.md]: **nitpick:** 

```suggestion
## Acceptance criteria (Informal)
```

**Review by tancnle — CHANGES_REQUESTED** [review]
Thanks, @j-6. I have left some suggestions for your consideration. Please let me know what you think.

Is this PR ready for review? I noted that this is still set as draft 🤔

**Comment by j-6** [comment]
Thank you @tancnle, I've made those changes. I'm going to keep this PR open if that's OK, I would like my team to add to this PR with other project epics.

**Comment by tancnle** [comment]
> Thank you @tancnle, I've made those changes. 

Thanks @j-6 👍🏼 

> I'm going to keep this PR open if that's OK, I would like my team to add to this PR with other project epics.

I would suggest keeping this PR as small as possible and merging. This work has been reviewed. Your team can create separate PR per project epic. What do you think?

**Review by tancnle — APPROVED** [review]

**Pull request merged**